spec/helper.js

```diff
@@ -217,11 +217,12 @@ global.helper = {
      * @returns {Promise}
      */
     stop: function () {
-        if (this.core) {
-            return this.core.stop();
-        } else {
-            return bluebird.resolve();
-        }
+      var promises = [new Promise(function (resolve) {
+        setTimeout(resolve);
+      })];
+      if (this.core) promises.push(this.core.stop());
```

Comment:
I think we shy away from one line ifs in our general, unwritten style conventions.